socialdistribution/app/views.py

```diff
@@ -426,12 +426,27 @@ def sync_github_activity(request):
 
 def github_event_to_post_adapter(author, event):
     type = Post.ContentType.PLAIN
-    title = f"Github Event of type {event['type']}"
+    title = f"Github: {event['type']}"
     categories = f"github, {event['type']}"
+    try:
+        head = str(event['payload']['head'])
+    except:
+        head = "N/A"
+    try:
+        messages = []
+        for commit in event['payload']['commits']:
+            messages.append(commit['message'])
+        messages = ", ".join(messages)
+    except:
```

Comment:
Same as previous

Reply:
Thanks for the advice, I have changed it to except KeyError.